libr/util/Makefile

```diff
@@ -37,6 +37,14 @@ endif
 
 include deps.mk
 
+SDBCHARSETPATH=$(DESTDIR)$(DATADIR)/radare2/$(VERSION)/util
+CWD=$(shell pwd)
+
+all:
+	mkdir -p "${SDBCHARSETPATH}"
```

Comment:
unnecessarsy line. the line below is using -p so this directory will be also created